pkgs/servers/web-apps/funkwhale/default.nix

```diff
@@ -0,0 +1,42 @@
+{ pkgs ? import <nixpkgs> {} }:
+with pkgs;
```

Comment:
`import <nixpkgs> {}` is disallowed in nixpkgs because it's impure. Take dependencies as arguments, look at the other packages and read up on callPackage.